pytest_remotedata/plugin.py

```diff
@@ -31,7 +34,11 @@ def pytest_configure(config):
 
     strict_check = bool(config.getini('remote_data_strict'))
 
+    if config.option.remote_data_only and config.option.remote_data != 'any':
+        config.option.remote_data = 'any'
```

Comment:
Is this really the behavior we want or do we still want to support narrowing to specific data sources even when running with `--remote-data-only`?

Reply:
Also maybe we want to check for invalid user input like providing both `--remote-data=none` and `--remote-data-only`